iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractTransformerXml.java

```diff
@@ -164,18 +172,23 @@ protected static void writePhaseTapChanger(String name, PhaseTapChanger ptc, Net
 
     protected static void readPhaseTapChanger(String name, PhaseTapChangerAdder adder, Terminal terminal, NetworkXmlReaderContext context) throws XMLStreamException {
         int lowTapPosition = XmlUtil.readIntAttribute(context.getReader(), ATTR_LOW_TAP_POSITION);
-        int tapPosition = XmlUtil.readIntAttribute(context.getReader(), ATTR_TAP_POSITION);
-        boolean regulating = XmlUtil.readOptionalBoolAttribute(context.getReader(), ATTR_REGULATING, false);
-        double targetDeadband = readTargetDeadband(regulating, context);
-        PhaseTapChanger.RegulationMode regulationMode = PhaseTapChanger.RegulationMode.valueOf(context.getReader().getAttributeValue(null, "regulationMode"));
+        Integer tapPosition = XmlUtil.readOptionalIntegerAttribute(context.getReader(), ATTR_TAP_POSITION);
+        String regulationModeStr = context.getReader().getAttributeValue(null, "regulationMode");
```

Comment:
Also it would be nice to have a XmlUtil.readOptionalEnumAttribute(reader, PhaseTapChanger.RegulationMode.class)